phases/account.go

```diff
@@ -1,9 +1,106 @@
 package phases
 
-import "fmt"
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"net/http"
 
-// Account ...
-func Account() (string, error) {
-	fmt.Println("SetAccount")
-	return "", nil
+	"github.com/bitrise-io/go-utils/log"
+)
+
+type organizationData struct{
```

Comment:
You are using this structure to have two fields, one of the fields at least is unique. This type of structure matches to the built-in map type. Use map instead.

Reply:
Also the code base not using any benefits of having a struct.